rocketpy/simulation/flight.py

```diff
@@ -697,9 +697,8 @@ def __init__(
                     parachute.noisy_pressure_signal.append([node.t, pressure + noise])
                     # Gets height above ground level considering noise
                     hAGL = (
-                        self.env.pressure.find_input(
+                        self.env.barometric_height(
                             pressure + noise,
```

Comment:
Remove the leading comma and run black again
```suggestion
                            pressure + noise
```